### PR TITLE
DataGrid - Add scrollbar spacing in footer with native scrolling and fixed columns (T846658)

### PR DESCRIPTION
**Pull request by igormalzew**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js [hunk 1]
```diff
@@ -575,6 +575,58 @@ function createGridView(options, userOptions) {
...
+
+        const $container = $('#container');
+        $container.height(110).width(1000);
+
+        gridView.render($container, $.extend(this.options, {
+            scrolling: {
+                useNative: true
+            },
+            showColumnHeaders: true,
+            groupPanel: {
```
Comment (vconst) [inline testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js]: Why groupPanel and searchPanel are enabled in this test?

File: testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js [hunk 2]
```diff
@@ -575,6 +575,58 @@ function createGridView(options, userOptions) {
...
+                    { values: [2] },
+                    { values: [3] },
+                    { values: [4] },
+                    { values: [5] }
+                ],
+                totalItem: {
+                    summaryCells: [
+                        { summaryType: 'count', value: 100 },
+                        { summaryType: 'min', value: 0 },
+                        { summaryType: 'max', value: 120001 }
```
Comment (vconst) [inline testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js]: Why is three summaryCells here?

File: testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js [hunk 3]
```diff
@@ -575,6 +575,58 @@ function createGridView(options, userOptions) {
...
+        }));
+
+        gridView.update();
+        gridView.getController('resizing').resize();
+
+        // assert
+        const footerTable = $container.find('.dx-datagrid-content-fixed').eq(0);
+        const scrollerWidth = gridView.getView('rowsView').getScrollbarWidth();
+
+        assert.equal(parseFloat(footerTable.css('paddingRight')), scrollerWidth, 'footer has padding');
```
Comment (vconst) [inline testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js:619]: Check that vertical scroll is exists (content height is greater than container height)

**Review by vconst — CHANGES_REQUESTED** [review]

File: testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js (hunk 3 above)
Comment (igormalzew) [reply testing/tests/DevExpress.ui.widgets.dataGrid/gridView.tests.js:619]: Done

**Review by Alyar666 — APPROVED** [review]

**Review by vconst — APPROVED** [review]

**Pull request merged**